src/main/java/seedu/address/logic/commands/DeassignmentIndivCommand.java

```diff
@@ -0,0 +1,113 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Assignment;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+
+/**
+ * Unmarks the attendance of an existing student in the list.
+ */
+public class DeassignmentIndivCommand extends Command {
+    public static final String COMMAND_WORD = "deassignIndiv";
+    public static final String MESSAGE_SUCCESS = "Assignment deassigned successfully!";
+    public static final String MESSAGE_FAIL = "Assignment not found.";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deassigns an assignment for one student.\n"
+            + "Compulsory Parameters: "
+            + "INDEX (must be a positive integer) "
+            + PREFIX_NAME + "ASSIGNMENT_NAME "
+            + "\nExample: " + COMMAND_WORD + " 1 "
+            + PREFIX_NAME + "Tutorial1 ";
+    private final Name name;
+
+    private final Index index;
+
+    /**
+     * Constructs an DeassignmentIndivCommand with the specified name and maximum score.
+     *
+     * @param index of the person in the filtered person list to edit
+     * @param name The name of the assignment. Must not be null.
+     * @throws NullPointerException If the provided name is null.
+     * @throws IllegalArgumentException If the provided maxScore is not greater than 0.
```

Comment:
This is an inaccurate documentation, `IllegalArgumentException` is never thrown here.